gem/static/js/springster.js

```diff
@@ -7,6 +7,11 @@
       document.readyState === "interactive" || document.readyState === "complete" ? callback() : document.addEventListener("DOMContentLoaded", callback);
   };
 
+  var remNoJS = function() {
```

Comment:
@projectmushroom display:none; removes the element on the DOM, does this kick in before that?

Is there a difference between 
```
<noscript>
      <style>#more-link {display:none;}</style>
 </noscript>
```

// Toggle loadmore button on js freindly featurephones
```
.no-js .more-link {
  display: none;
}
```